vault_service/views/user.py

```diff
@@ -195,4 +197,451 @@ def store_data():
         return json.dumps(data), 200
 
 
+@advertise(scopes=['store-myads'], rate_limit=[1000, 3600*24])
```

Comment:
This is a new scope, so when deployed, none of our users will be able to use these endpoints. We would need to update the database (for all non-anonymous users) and change adsws config: https://github.com/adsabs/BeeHive/blob/00730df6d0582db59b7932a1567393c46df2bccf/images/common/adsws/local_config.py#L97